### DISCUSSION TITLE
Feature: Google Drive Support

### DISCUSSION THREAD

**Discussion by robertofgama** [aidenlx/media-extended#287] [Website]
Hello, first thanks for your plugin, you work helps me a lot. 
Second i would like to sugest that your plugin support videos hosted on google drive, i have a lot of MP4 videos on my google drive account would be wanderful if they work like youtube videos.

**Comment by the-ethernal** [comment]
I'd find this useful too! c:

**Comment by shuhanliu787** [comment]
or is there any way i can upload my local video to some drive and open it in obsidian, since i cannot keep all my video in my computer locally
